server/channels/store/sqlstore/draft_store.go

```diff
@@ -181,6 +181,27 @@ func (s *SqlDraftStore) Delete(userID, channelID, rootID string) error {
 	return nil
 }
 
+func (s *SqlDraftStore) PermanentDeleteByUser(userID string) error {
+	query := s.getQueryBuilder().
+		Delete("Drafts").
+		Where(sq.Eq{
+			"UserId": userID,
+		})
+
+	sql, args, err := query.ToSql()
+	if err != nil {
+		return errors.Wrapf(err, "PermanentDeleteByUser: failed to generate SQL query for permanently deleting drafts by user")
+	}
+
+	_, err = s.GetMaster().Exec(sql, args...)
```

Comment:
Use `ExecBuilder` directly.

Reply:
@Aryakoste can you do it? Everything else is excellent and looks ready to merge after this.